UI/src/locales/de.json

```diff
@@ -0,0 +1 @@
":[null,"Belegliste"],"Vouchers":[null,"Gutschein"],"Wages and Deductions":[null,"Löhne und Abzüge"],"Wages/Deductions":[null,"LÃ¶hne/Absetzungen"],"Warehouse":[null,"Warenlager"],"Warehouse deleted!":[null,""],"Warehouse saved!":[null,""],"Warehouses":[null,"Warenlager"],"Warning!":[null,"Warnung!"],"Wed":[null,"Mit"],"Week":[null,"Woche"],"Week Starting":[null,"Woche begonnen am"],"Week(s)":[null,"Woche(n)"],"Weekly Time and Materials Entry":[null,""],"Weeks":[null,"Wochen"],"Weight":[null,"Gewicht"],"Weight Unit":[null,"Gewichtseinheit"],"What type of item is this?":[null,"Welche Artikelart ist es?"],"Where shall we send the backup?":[null,"Wohin sollen wir die Datensicherung schicken?"],"Whole Month Straight Line":[null,"Ganzer Monat geradlinig"],"Widgit Themes":[null,"Widgit Themen"],"Work Order":[null,"Arbeitsblatt"],"Would you like to migrate the database?":[null,""],"Would you like to upgrade the database?":[null,""],"XLS":[null,"XLS"],"XLSX":[null,""],"Yahoo":[null,"Yahoo"],"Year":[null,"Jahr"],"Year End":[null,"Jahresabschluss"],"Year(s)":[null,"Jahr(e)"],"Yearend":[null,"Jahresabschluss"],"Yearend complete":[null,"Jahresabschluss beendet"],"Years":[null,""],"Yes":[null,"Ja"],"ZIP/Post Code":[null,"Postleitzahl"],"Zero":[null,"Null"],"Zip Code":[null,"Postleitzahl"],"Zip/Post Code":[null,"Postleitzahl"],"Zip/Post Code:":[null,"Postleitzahl:"],"Zip/Postal Code":[null,""],"[_1]\n[_2]":[null,"[_1]\n[_2]"],"[_1] Cash Transfer Transaction":[null,"[_1] BarÃ¼bertragung buchen"],"[_1] Credit Note":[null,"[_1] Gutschrift"],"[_1] Debit Note":[null,"[_1] Lastschrift"],"[_1] General Ledger Transaction":[null,"[_1] HinzufÃ¼gen einer Buchung zum Hauptbuch"],"[_1] Number":[null,"[_1] Nummer"],"[_1] [_2] Transaction":[null,""],"[_1]: Customer not on file!":[null,"[_1]: Kunde nicht in der Kartei!"],"[_1]: Vendor not on file!":[null,"[_1]: Lieferant nicht in der Kartei!"],"action not defined!":[null,"Aktion nicht definiert!"],"balance":[null,"Balance"],"city":[null,"Stadt"],"csv":[null,""],"days":[null,"Tage"],"debits":[null,"Belastungen"],"default":[null,""],"delete":[null,"lÃ¶schen"],"depreciation":[null,""],"disposal":[null,""],"done":[null,"Erledigt"],"e":[null,""],"ea":[null,"ea"],"failed":[null,"fehlgeschlagen"],"from_date":[null,""],"html":[null,""],"import":[null,"importieren"],"in months":[null,"in Monaten"],"in use":[null,""],"in years":[null,"in Jahren"],"is bigger than the amount available":[null,"ist grÃ¶sser als der verfÃ¼gbare Betrag"],"is lesser than 0":[null,"ist kleiner als 0"],"is lesser than the amount to be paid":[null,"ist weniger als der zu zahlende Betrag"],"is null":[null,"ist null"],"label":[null,""],"partial disposal":[null,"teilweisige Entsorgung"],"permalink":[null,"Permalink"],"production":[null,"Produktion"],"sent":[null,"gesendet"],"tex":[null,""],"time":[null,"Zeit"],"time(s)":[null,"Zeit(en)"],"to_date":[null,""],"unexpected error!":[null,"unerwarteter Fehler!"],"use of an overpayment":[null,"Nutzen einer Ãœberzahlung"]}
```

Comment:
in `{"30":[null,"30"],...}` what is the `null` about?

Reply:
Leftover from full json format. Converted to standard.